packages/strapi-plugin-users-permissions/admin/src/components/ListRow/index.js

```diff
@@ -32,19 +34,19 @@ class ListRow extends React.Component { // eslint-disable-line react/prefer-stat
         icoType: 'pencil',
         onClick: this.handleClick,
       },
-      {
-        icoType: 'trash',
-        onClick: () => { this.setState({ showModalDelete: true }); },
-      },
+      // {
```

Comment:
It means that the delete icon won't available? We have to be able to remove a role.